docs/operator-manual/troubleshooting.md

```diff
@@ -82,15 +82,15 @@ kubectl exec -n argocd -it \
   $(kubectl get pods -n argocd -l app.kubernetes.io/name=argocd-application-controller -o jsonpath='{.items[0].metadata.name}') bash
 ```
 
-2 Use `argocd-util cluster kubeconfig` command to export kubeconfig file from the configured Secret:
+2 Use `argocd-util kubeconfig` command to export kubeconfig file from the configured Secret:
```

Comment:
This command is part of recent change https://github.com/argoproj/argo-cd/issues/5690. It will be available in v2.0.

Reply:
If you are using argocd v1.8 then please use corresponding document version https://argo-cd.readthedocs.io/en/release-1.8/operator-manual/troubleshooting/